scripts/sort_sparse_matrix.py

```diff
@@ -0,0 +1,75 @@
+#! /usr/bin/env python
+"""Sort sparse expression matrix files by gene, barcode
```

Comment:
It would be good to have an explanation for what this file is intended for -- i.e. it is NOT used in ingest-pipeline sorting, but is just a helper